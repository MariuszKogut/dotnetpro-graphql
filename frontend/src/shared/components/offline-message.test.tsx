import React from 'react'
import { render } from '@testing-library/react'
import OfflineMessage from './offline-message'

describe('OfflineMessage', () => {
  it('renders without errors', () => {
    const { container } = render(<OfflineMessage />)
    expect(container).toMatchSnapshot()
  })
})
