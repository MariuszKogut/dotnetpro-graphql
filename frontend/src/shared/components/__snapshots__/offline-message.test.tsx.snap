// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfflineMessage renders without errors 1`] = `
<div>
  <div
    class="fade alert alert-warning show"
    role="alert"
  >
    Sie sind gerade offline. Es wird eine lokale Kopie der Daten gezeigt. Diese kann von den aktuellen Daten abweichen.
  </div>
</div>
`;
